### ISSUE TITLE
Remove zoho.com 

### ISSUE THREAD

**Issue by richardonrails** [micke/valid_email2#209]
zoho.com appears to be a real b2b mail service, it probably doesn't deserve to be on this list. One of my customers was blocked because their domain's MX record points to zoho.

**Comment by micke** [comment]
Sorry about that, i've pushed the 5.0.1 release to fix this!

**Comment by richardonrails** [comment]
Thanks! That fixed it.

However in the meantime I tried to work around the issue with a whitelist. I changed this:

`  validates :email, 'valid_email_2/email': { mx: !Rails.env.test?, disposable: true, blacklist: true }`

To this:
`  validates :email, 'valid_email_2/email': { mx: !Rails.env.test?, disposable_with_whitelist: true, blacklist: true }`

And added this file `config/whitelisted_email_domains.yml`
```
- zoho.com
```

However that didn't fix it in prod for my customer who's domain's MX points to zoho.

Did I do something wrong?
